docs/docs/behaviors/macros.md

```diff
@@ -62,6 +78,30 @@ bindings
 There are a set of special macro controls that can be included in the `bindings` list to modify the
 way the macro is processed.
 
+### Parameters
+
+When creating a macro that takes parameter(s), there are control when the parameters passed to the macro are used
```

Comment:
This sentence (specifically "there are control when the parameters [...]") seems to be broken, I couldn't tell the intention.

Reply:
This was... incomprehensible.... Remind me not to write docs late at night when I'm tired. Reworded, to hopefully actually make sense now. Thanks!